zebrad/tests/acceptance.rs

```diff
@@ -20,7 +20,91 @@
 //! If it does not have any IPv4 interfaces, IPv4 localhost is not on `127.0.0.1`,
 //! or you have poor network connectivity,
 //! skip all the network tests by setting the `ZEBRA_SKIP_NETWORK_TESTS` environmental variable.
-
+//!
+//! ## Large/full sync tests
+//!
+//! This file has sync tests that are marked as ignored because they take too much time to run.
+//! Some of them require environment variables or directories to be present.
+//! Here are some examples on how to run each of them:
+//!
+//! ```console
+//! $ cargo test sync_large_checkpoints_mainnet -- --ignored --nocapture
+//!
+//! $ cargo test sync_large_checkpoints_mempool_mainnet -- --ignored --nocapture
+//!
+//! $ sudo mkdir /zebrad-cache
+//! $ sudo chmod 777 /zebrad-cache
+//! $ export FULL_SYNC_MAINNET_TIMEOUT_MINUTES=600
+//! $ cargo test full_sync_mainnet -- --ignored --nocapture
+//!
+//! $ sudo mkdir /zebrad-cache
+//! $ sudo chmod 777 /zebrad-cache
+//! $ export FULL_SYNC_TESTNET_TIMEOUT_MINUTES=600
+//! $ cargo test full_sync_testnet -- --ignored --nocapture
+//! ```
+//!
+//! ### Notes
+//!
+//! - `FULL_SYNC_MAINNET_TIMEOUT_MINUTES` env variable: The total number of minutes we
+//! will allow this test to run or give up. Value for the Mainnet full sync tests.
+//! - `FULL_SYNC_TESTNET_TIMEOUT_MINUTES` env variable: The total number of minutes we
+//! will allow this test to run or give up. Value for the Testnet ful  sync tests.
+//! - `/zebrad-cache` directory: For some sync tests, this needs to be created in
+//! the file system, the created directory should have write permissions.
+//!
+//! Please refer to the documentation of each test for more information.
+//!
+//! ## Lightwalletd tests
+//!
+//! The lightwalletd software is an interface service that uses zebrad or zcashd RPC methods to serve wallets or other applications with blockchain content in an efficient manner.
+//! There are several versions of lightwalled in the form of different forks. The original
+//! repo is <https://github.com/zcash/lightwalletd> but Zecwallet Lite uses a custom fork: <https://github.com/adityapk00/lightwalletd>.
+//! The custom fork from adityapk00 is the one Zebra use for this tests:
+//!
+//! Zebra lightwalletd tests are not all marked as ignored but none will not run unless
+//! at least the `ZEBRA_TEST_LIGHTWALLETD` environment variable is present.
+//! Here are some examples of running each test:
+//!
+//! ```console
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ cargo test lightwalletd_integration -- --nocapture
```

Comment:
Reliability: The scripts here work right now, but there is no guarantee they will continue to work as we make changes to the tests.

And we might forget to update this documentation when we add new tests.

Reply:
this comment is not clear

Reply:
My understading is: the way we call these tests is by calling `entrypoint.sh`.

If we add the explicit command here, and later change `entrypoint.sh`, then they could become out of sync.

One alternative would be to change the command above, from an explicit `cargo test`, to a call to `entrypoint.sh`. I wonder if that would be a bit confusing though.

So my take is that this is fine to leave as is for now, and we can improve it later if it causes problems.

Reply:
Thanks, lets leave it as it is by now.